package.json

```
@@ -41,7 +41,7 @@
     "federalist": "npx gulp build",
     "lint": "npx gulp lintJS lintSass && npm run prettier:scss",
     "prestart": "npx gulp build",
-    "proof": "bundle exec htmlproofer --allow-hash-href=true --enforce-https=false --allow-missing-href=true --ignore-empty-alt=true --ignore-status-codes 0,403,429,503,302 --swap-urls 'https\\://designsystem.digital.gov/:/' --ignore-urls '/github.com/uswds/uswds/issues/' --ignore-files \"//select-a-language//,//2017//,//2018//,//2019//,//code-guidelines//\" ./_site",
+    "proof": "bundle exec htmlproofer --enforce-https=false --allow-missing-href=true --ignore-status-codes 0,403,429 --swap-urls 'https\\://designsystem.digital.gov/:/' --ignore-urls '/github.com/uswds/uswds/issues/' --ignore-files '/select-a-language/,/2017/,/2018/,/code-guidelines/' ./_site",
```

Hi @mahoneycm, I was hoping we could remove the `/select-a-language` directory from the "ignore files" config here as well so that we don't miss any more broken links on these pages. 

I did a little bit of poking around and found that the related nokogiri error goes away when I remove the `#page=16` hash on this link on the [Preferred language page](https://designsystem.digital.gov/patterns/select-a-language/language-preferences/#references): `https://www.whitehouse.gov/wp-content/uploads/legacy_drupal_files/omb/memoranda/2017/m-17-06.pdf#page=16`. Not quite sure why this removes the error, but curious if we could find a way to edit this link and remove that directory from the ignored files list. Curious what you think!

```suggestion
    "proof": "bundle exec htmlproofer --enforce-https=false --allow-missing-href=true --ignore-status-codes 0,403,429 --swap-urls 'https\\://designsystem.digital.gov/:/' --ignore-urls '/github.com/uswds/uswds/issues/' --ignore-files '/2017/,/2018/,/code-guidelines/' ./_site",
```

---

Thank you SO much for finding this! It was a big thorn in my side. I'll do some additional poking and prodding. I'm assuming it's because this links a to a PDF and the page number is set using a hash but there is no associated heading level like there would be for a webpage

---

@amyleadem ok so I opted to remove the page number. I feel like removing the page number is ideal so that we can continue to proof the link. 

I was able to successfully ignore just this link though if we want to continue to link to that specific page, we could instead update the script to ignore it.

I also tried alternative means of creating a jump link to that section but it doesn't look like we can without editing the actual PDF.

---

Resolved by [ba50af9](https://github.com/uswds/uswds-site/pull/2588/commits/ba50af9fcde967b2d16aaf2b609d3ce4688bcbd1)